frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/RelativeDatePicker.tsx

```diff
@@ -36,17 +36,9 @@ type Props = {
   reverseIconDirection?: boolean;
 };
 
-export const PastPicker = (props: Props) => (
-  <RelativeDatePicker {...props} formatter={value => value * -1} />
-);
-
-export const NextPicker = (props: Props) => (
-  <RelativeDatePicker
-    {...props}
-    offsetFormatter={value => value * -1}
-    reverseIconDirection
-  />
-);
+type OptionsContentProps = RelativeDatePickerProps & {
+  setOptionsVisible: (arg0: boolean) => void;
```

Comment:
Can you please come up with a more descriptive name for the parameter instead of `arg0`?